subprojects/gradle/deeplink-generator/src/gradleTest/kotlin/com/avito/deeplink_generator/ValidatePublicDeeplinksTest.kt

```diff
@@ -0,0 +1,142 @@
+package com.avito.deeplink_generator
+
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.gradlew
+import com.avito.test.gradle.module.AndroidLibModule
+import com.avito.test.gradle.plugin.plugins
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+internal class ValidatePublicDeeplinksTest {
+
+    @Test
+    fun `assemble single library - validate same links - success`(@TempDir projectDir: File) {
```

Comment:
https://english.stackexchange.com/a/235095

Reply:
Thanks for a hint, fixed